test/new-e2e/pkg/utils/common/output_dirs.go

```diff
@@ -0,0 +1,21 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package common
+
+import (
+	"strings"
+)
+
+// SanitizeDirectoryName replace invalid characters in a directory name underscores.
+//
+// Example:
+//   - name: TestInstallSuite/TestInstall/install_version=7.50.0
+//   - output directory: <root>/TestInstallSuite/TestInstall/install_version_7_50_0
+func SanitizeDirectoryName(name string) string {
```

Comment:
It might be safer to list allowed characters and replace everything else ?

Reply:
To be discussed, the implementation of this PR is using current code introduced by https://github.com/DataDog/datadog-agent/pull/29580. I'd like to have more context from the suite name in the folder's name, so this might change. I agree with you, we might simplify the logic - the fact that go test names match their structure, and that they accidentally match a folder's tree, should not mandate how we name suite's folders.